connector/connect/server/src/test/scala/org/apache/spark/sql/connect/ProtoToParsedPlanTestSuite.scala

```diff
@@ -115,7 +116,7 @@ class ProtoToParsedPlanTestSuite extends SparkFunSuite with SharedSparkSession {
       val planner = new SparkConnectPlanner(spark)
       val catalystPlan =
         analyzer.executeAndCheck(planner.transformRelation(relation), new QueryPlanningTracker)
-      val actual = normalizeExprIds(catalystPlan).treeString
+      val actual = normalizeExprIds(ReplaceExpressions(catalystPlan)).treeString
```

Comment:
@hvanhovell is there a specific reason to replace `RuntimeReplaceable` in the golden files? It makes the golden file a bit hard to read as there are quite some expressions that will be replaced with `StaticInvoke`/`Invoke` and you will see something like
```
Project [staticinvoke(class org.apache.spark.sql.catalyst.expressions.ExpressionImplUtils, BinaryType, aesEncrypt, cast(g#0 as binary), cast(g#0 as binary), g#0, DEFAULT, cast( as binary), cast( as binary), BinaryType, BinaryType, StringTypeAnyCollation, StringTypeAnyCollation, BinaryType, BinaryType, true, true, true) AS aes_encrypt(g, g, g, DEFAULT, , )#0]
```

Reply:
Also for testing, I think it's good enough to verify the analyzed plan of Spark Connect.

Also cc @HyukjinKwon